packages/lodestar/src/tasks/index.ts

```diff
@@ -32,8 +36,11 @@ export class TasksService implements IService {
   private readonly sync: IBeaconSync;
   private readonly network: INetwork;
   private readonly logger: ILogger;
+  private readonly reps: IReputationStore;
 
   private interopSubnetsTask: InteropSubnetsJoiningTask;
+  private diversifyPeersTask: DiversifyPeersBySubnetTask;
+  private checkPeerAliveTask: CheckPeerAliveTask;
```

Comment:
how about having them in subfolder of `tasks` like `tasks/network`? when I want to audit all routines we have I can just look at this

would like to hear others ideas too @wemeetagain @dapplion @3xtr4t3rr3str14l , to see if we want `tasks/network` or `network/tasks`

Reply:
I agree with @mpetrunic's thoughts on these tasks being things that require multiple components.

I think these could be candidates here tho, both rely on sync completion to start. Imo this is fine for now.
I think one possibility would be to introduce a module for managing validator resources, handling any additional resources required for attesting/proposing. It would track connected validators, manage eth1data fetching/retrieving, manage subnet subscriptions, etc. All things that a simple read-only beacon chain doesn't require.

Reply:
Peer liveness and metadata tracking is just related to networking and should live there